pinot-query-planner/src/test/resources/queries/GroupByPlans.json

```diff
@@ -91,6 +91,92 @@
           "\n                  LogicalTableScan(table=[[a]])",
           "\n"
         ]
+      },
+      {
+        "description": "SQL hint based group by optimization with select and aggregate column",
```

Comment:
From my understanding of the code this hint can potentially be used for the following types of queries in addition to the group by queries tested here:

- Select only aggregation columns with group-by (e.g. `SELECT SUM(a.col3) FROM a GROUP BY a.col3`)
- Select only aggregation columns without group by (e.g. `SELECT SUM(a.col3) FROM a`)

Do you intend this hint to act on the above type of queries?
If so, can you add some test cases for the above?

Reply:
I don't think this hint is applicable to the 2nd type of queries unless I am missing something.

Reply:
Good catch. This is needed only for group by aggregates. Fixed it.